Gopkg.lock

```diff
@@ -173,6 +173,7 @@
     "github.com/dgrijalva/jwt-go",
     "github.com/goenning/letteravatar",
     "github.com/goenning/sqlcertcache",
+    "github.com/google/uuid",
```

Comment:
Just revert all changes to Gopkg.lock

Reply:
ah.. thought I reverted all changes. will update and squash commits or create a new PR with one commit